src/main/io/osd.h

```diff
@@ -118,6 +119,7 @@ typedef enum {
     OSD_STAT_TIMER_2,
     OSD_STAT_MAX_SPEED,
     OSD_STAT_MAX_DISTANCE,
+    OSD_STAT_TOTAL_DISTANCE,
```

Comment:
New stats should be added at the end.  See the comment directly above the enumeration.
```
// *** IMPORTANT ***
// The order of the OSD stats enumeration *must* match the order they're displayed on-screen
// This is because the fields are presented in the configurator in the order of the enumeration
// and we want the configuration order to match the on-screen display order.
// Changes to the stats display order *must* be implemented in the configurator otherwise the
// stats selections will not be populated correctly and the settings will become corrupted.
//
// Also - if the stats are reordered then the PR version must be incremented. Otherwise there
// is no indication that the stored config must be reset and the bitmapped values will be incorrect.
```

Also since each stat is stored with a bit position changing the order will corrupt existing configurations.